internal/bminventory/inventory.go

```diff
@@ -311,6 +354,57 @@ func (b *bareMetalInventory) getUserSshKey(params installer.GenerateClusterISOPa
 		"groups": [ "sudo" ]}`, sshKey)
 }
 
+func validateIgnitionConfig(ignition string) error {
```

Comment:
i think we have ignition.go file. Maybe it can be better place?

Reply:
Yeah, that's a good idea. I don't think that file existed when I wrote this initially :laughing: